.github/actions/build-docs/action.yml

```diff
@@ -0,0 +1,13 @@
+# Use a step like this to build documentation.
```

Comment:
This template text could be removed. (I will remove it from my copy as well.)

Reply:
Fixed